tutorials/Consistent_Scores.ipynb

```diff
@@ -0,0 +1,465 @@
+    "\n",
+    "\n",
+    "You may already know that the MSE is a consistent score for forecasting the mean and the MAE is a consistent score for forecasting the median. But did you know that there are a whole family of scores that are consistent for prediciting the mean, median, or quantiles? \n",
+    "\n",
+    "The consistent scoring module in `scores` provides access to these families of consistent scores hat can be used to tailor emphasis of predictive performance based on decision thresholds of interest.\n",
+    "\n",
+    "### Example 1. A score consistent with the mean\n",
+    "Let's jump into an example that illustrates a scoring function that is consistent with predicting the mean value using `scores`. Suppose we want to evaluate the performance of expected daily precipitation forecasts, but we want to place increasing importance on correctly forecasting extreme values. \n",
+    "\n",
+    "The mean (or expectation) functional is equivalent to the 0.5-[expectile](https://en.wikipedia.org/wiki/Expectile), so we want to use the `consistent_expectile_score` in `scores` to evaluate the expected daily precipitation forecasts. Note that the mean is identical to the 0.5-expectile in the same way that the median is identical to the 0.5-quantile (or 50th percentile).\n",
+    "\n",
+    "A scoring function $S$ that is consistent for the alpha-expectile takes the following form ([Savage 1971](https://doi.org/10.2307/2284229)):\n",
+    "\n",
+    " $$ S(x, y) =\n",
+    "    \\begin{cases}\n",
+    "    (1 - \\alpha)(\\phi(y) - \\phi(x) - \\phi'(x)(y-x)), & y < x \\\\\n",
+    "    \\alpha(\\phi(y) - \\phi(x) - \\phi'(x)(y-x)), & x \\leq y\n",
+    "    \\end{cases}\n",
+    "$$\n",
+    "\n",
+    "Where:\n",
+    "- $x$ is the forecast\n",
+    "- $y$ is the observation\n",
+    "- $\\alpha$ is the expectile level\n",
+    "- $\\phi$ is a [convex function](https://en.wikipedia.org/wiki/Convex_function) of a single variable\n",
+    "- $\\phi'$ is the [subderivative](https://en.wikipedia.org/wiki/Subderivative) of $\\phi$. The subderivative is a generalisation of the derivative for convex functions and coincides with the derivative when the convex function is differentiable\n",
+    "- $S(x,y)$ is the score.\n",
+    "\n",
+    "In this case, we choose $\\alpha=0.5$, since the 0.5-expectile is identical to the mean.\n",
+    "\n",
+    "If one wants to control the relative importance of predictive performance across different decision thresholds, they must first create a weighting function $\\phi''$ which is the second derivative of $\\phi$.\n",
+    "\n",
+    "Let's assume that the importance of accurate prediction increases exponentially with increasing decision threshold $z$. We then need to determine our $\\phi(z)$ and $\\phi'(z)$ functions. \n",
+    "\n",
+    "First we create the weighting function\n",
```

Comment:
Consider whether to include a graph showing the weighting function, e.g. 
<img width="632" alt="image" src="https://github.com/nci/scores/assets/134973832/d633efb3-4f99-4c41-9d25-35617adf069a">
(generated using desmos.com, I'd suggest integrating it into the notebook rather than using a screenshot)

Reply:
I have produced a plot of this weighting function.